advocacy_docs/edb-postgres-ai/migration-etl/data-migration-service/terminology.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: "Terminology"
+description: Learn some basic concepts associated with EDB Data Migration Service. 
+---
+
+This terminology is important to understand EDB Data Migration Service (DMS) functionality.
+
+## Analytics Sync
+
+EDB Postgres® AI Analytics Sync is a type of replication/migration supported by EDB Data Migration Service. EDB Postgres Advanced Server and Postgres source database snapshots are transformed into Delta Lake format in CSP Object Storage. This object storage is exposed as Storage Locations in the EDB Postgres AI Console.
+
+## Change Data Capture (CDC)
+
+CDC is a set of software design patterns used to determine and track changes in data sets by calculating *deltas*. EDB Data Migration Service uses CDC to stream changes from a database cluster to another. 
+
+## EDB DMS Reader
```

Comment:
@mlewandowski-edb  
As a short term solution, I nested the Kafka and Debezium entries under EDB DMS Reader so they are not as "prominent". I also added a sentence to each entry explaining very broadly what their functions are to DMS. 

We can still delete the entries for Kafka and Debezium if desired, but we do mention Debezium a couple of times in the documentation, so I am not sure this is a good idea. Maybe instead of deleting, we can provide a couple more sentences about what their functions are in DMS?

CC: @ksnavely @jperozoa